### PR TITLE
Port reference document

### PR DESCRIPTION
**Pull request by ned1313**
Adding a port reference document with the ports used by all pods and services in Gloo Gateway and Gloo Enterprise.

### CODE REVIEW

**Comment by arianaw66** [comment]
oh whoops /skip-ci

File: docs/content/reference/gloo_port_reference.md [hunk 1]
```diff
@@ -0,0 +1,152 @@
+---
+title: "Gloo Port Reference"
+description: Listing of ports used by Gloo Gateway and Gloo Enterprise
+weight: 35
+---
+
+Gloo Gateway and Gloo Enterprise both deploy containers that listen on certain ports for incoming traffic. This document lists out the pods and services that make up Gloo and Gloo Enterprise, and the ports which these pods and services listen on. It is also possible to set up mutual TLS (mTLS) for communication between Gloo resources. The addition of mTLS changes the ports and traffic flows slightly, which is addressed in this document as well.
+
+This document is specific to the pods and service deployed in a Kubernetes environment. Deploying Gloo using the HashiCorp stack is also supported, and the port mappings and services should be the same.
```
Comment (arianaw66) [inline docs/content/reference/gloo_port_reference.md:9]:
```suggestion
This document is specific to the pods and services deployed in a Kubernetes environment. Deploying Gloo using the HashiCorp stack is also supported, and the port mappings and services should be the same.
```

File: docs/content/reference/gloo_port_reference.md [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+
+{{% notice note %}}
+It is possible to customize some port settings by providing custom values to the Helm chart that installs Gloo OSS and Gloo Enterprise. The port reference below is for an installation of Gloo that uses the default settings in the Helm chart.
```
Comment (arianaw66) [inline docs/content/reference/gloo_port_reference.md:12]: Is OSS a well-known acronym? Maybe we should just drop it or spell it out.

Comment (ashleywang1) [reply docs/content/reference/gloo_port_reference.md:12]: Open Source, I think it's used like that in other places in the gloo docs.

File: docs/content/reference/gloo_port_reference.md [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+{{% /notice %}}
+
+---
+
+## Gloo Gateway OSS
+
+Gloo open-source is the free, open-source version of Gloo Gateway. The installation process uses a Helm chart to create the necessary custom resource definitions (CRDs), deployments, services, pods, etc. The services and pods listen on specific ports to enable communication between the components that make up Gloo and outside sources that will consume Upstream resources through Gloo.
```
Comment (arianaw66) [inline docs/content/reference/gloo_port_reference.md:19]: Same idea here. Let's just add "software" for clarity.
```suggestion
Gloo open-source software is the free, open-source version of Gloo Gateway. The installation process uses a Helm chart to create the necessary custom resource definitions (CRDs), deployments, services, pods, etc. The services and pods listen on specific ports to enable communication between the components that make up Gloo and outside sources that will consume Upstream resources through Gloo.
```

File: docs/content/reference/gloo_port_reference.md (hunk 2 above)
Comment (ned1313) [reply docs/content/reference/gloo_port_reference.md:12]: Replaced OSS with Open-source.

**Pull request merged**